charts/flagger/templates/prometheus.yaml

```diff
@@ -201,6 +201,35 @@ data:
         source_labels:
         - __meta_kubernetes_pod_name
         target_label: kubernetes_pod_name
+
+    # scrape config for Kuma dataplanes
+    - job_name: 'kuma-dataplanes'
+      scrape_interval: "5s"
+      relabel_configs:
+        - source_labels:
+            - k8s_kuma_io_name
+          regex: "(.*)"
+          target_label: pod
+        - source_labels:
+            - k8s_kuma_io_namespace
+          regex: "(.*)"
+          target_label: namespace
+        - source_labels:
+            - __meta_kuma_mesh
+          regex: "(.*)"
+          target_label: mesh
+        - source_labels:
+            - __meta_kuma_dataplane
+          regex: "(.*)"
+          target_label: dataplane
+        - source_labels:
+            - __meta_kuma_service
+          regex: "(.*)"
+          target_label: service
+        - action: labelmap
+          regex: __meta_kuma_label_(.+)
+      kuma_sd_configs:
```

Comment:
This seems to crash Prometheus on clusters without Kuma, see e2e results.

Reply:
Hmm weird, will check this out and fix.

Reply:
Maybe we need to update the Prometheus version that we're using?

Reply:
Ye bumping prom to latest (2.32.1) works fine (it spits an warning to the logs because the kuma control plane doesn't exist) but it doesn't crash / fail. I also wonder whether we even need this in the prometheus config. We ship our own 'bundle' of yaml for prometheus with all the configs / dashboards (similiar to istio) and use Flagger's default of _not_ installing it's own prometheus. So maybe I can just remove this config from the default Flagger prom install and document that a user needs to add it themselves if _not_ using our (Kuma's) metrics bundle?

Reply:
Oh I wasn't aware that Kuma ships with Prometheus. In this case please remove the config from here. The docs should tell users to install Kuma with Prometheus is the same way we do for Istio, Linkerd, OSM, etc.

Reply:
Done